sql/database.sql

```diff
@@ -6562,12 +6562,30 @@ CREATE TABLE `lists` (
   `severity_al` VARCHAR( 50 ) DEFAULT NULL,
   `external_id` VARCHAR(20) DEFAULT NULL,
   `list_option_id` VARCHAR(100) DEFAULT NULL COMMENT 'Reference to list_options table',
+  `drug_dosage_instructions` longtext DEFAULT NULL COMMENT 'Medication list dosage instructions',
```

Comment:
make text fields bare (ok for the COMMENT, though); just trying to keep the text fields consistent since have spent resources in past standardizing these.